index.html

```diff
@@ -99,16 +112,23 @@ <h2>Multi-Post <br> Stories</h2>
 
             <ul class="tech-skills">
               <li>html</li>
+              <li class="desktop1">Ruby on rails</li>
               <li>css</li>
               <li>javaScript</li>
           </ul>
-          <button type="button">See Project</button>
+          <button class="btn1" type="button">See Project</button>
+         </div>
+         <img src="./Images/SnapshootPortfolio-desktop2.png" alt="SnapshootPortfolio-desktop2" class="img2">
+         <img src="./Images/SnapshootPortfolio1.png" alt="SnapshootPortfolio1" class="img3">
         </div>
 
         <div class="card">
-            <img src="./Images/SnapshootPortfolio2.png" alt="SnapshootPortfolio2">
+            <img src="./Images/SnapshootPortfolio2.png" alt="SnapshootPortfolio2" class="img1">
+            <img src="./Images/SnapshootPortfolio-desktop3.png" alt="SnapshootPortfolio-desktop3" class="img2">
+            <img src="./Images/SnapshootPortfolio2.png" alt="SnapshootPortfolio2" class="img3">
 
-            <h2>Multi-Post <br>Stories</h2>
+            <div>
+               <h2>Multi-Post <br>Stories</h2>
```

Comment:
- [ ]  You've done a good job, your project look nice, but its missing a very important feature, the responsiveness for screens that are above something around 1120px width,is not ok, the cards are getting fixed. 

Your design | Figma design
--- | ---
![image](https://user-images.githubusercontent.com/43877689/178114004-01c1b020-866e-4d8f-934c-3779f984b716.png)) | ![image](https://user-images.githubusercontent.com/43877689/178114043-be6aa6ef-a354-493e-90ff-d461ce4fa052.png)